integrated_channels/integrated_channel/migrations/0028_alter_contentmetadataitemtransmission_unique_together.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.2.19 on 2023-06-06 21:40
+
+from django.db import connection, migrations
+
+
+class Migration(migrations.Migration):
+    atomic = False
+    dependencies = [
+        ('integrated_channel', '0027_orphanedcontenttransmissions'),
+    ]
+    db_engine = connection.settings_dict['ENGINE']
+    if 'sqlite3' in db_engine:
+        operations = [
+            migrations.AlterUniqueTogether(
+                name='contentmetadataitemtransmission',
+                unique_together={('integrated_channel_code', 'plugin_configuration_id', 'content_id')},
+            ),
+        ]
+    else:
+        operations = [
+            migrations.SeparateDatabaseAndState(
+                state_operations=[
+                    migrations.AlterUniqueTogether(
+                        name='contentmetadataitemtransmission',
+                        unique_together={('integrated_channel_code', 'plugin_configuration_id', 'content_id')},
+                    ),
+                ],
+                database_operations=[
+                    migrations.RunSQL(sql="""
+                        ALTER TABLE integrated_channel_contentmetadataitemtransmission ADD CONSTRAINT
+                        integrated_channel_contentmetadataitemtransmission_integrated_channel_code_plugin_configuration_id_content_id_4a1f9a6a_uniq
```

Comment:
this index name is almost certainly too long for mysql57 - try installing this branch of edx-enterprise into devstack lms and run migrations, i bet you get an error

Reply:
looks like 64 char limit https://dev.mysql.com/doc/refman/5.7/en/identifier-length.html